.circleci/build_lib.sh

```diff
@@ -178,7 +178,7 @@ cmake $cmake_params ../lib-ledger-core
 echo "======> Build for $unamestr in $BUILD_CONFIG mode"
 if [ "$1" == "ios" ]; then
     echo " >>> Starting iOS build for architecture ${ARCH} with toolchain ${TOOLCHAIN_NAME} for ${OSX_SYSROOT}"
-    xcodebuild -project ledger-core.xcodeproj -configuration Release -jobs 4
+    xcodebuild -project ledger-core.xcodeproj -configuration Release -jobs 3
 else
-    make -j4
+    make -j3
```

Comment:
No sorry, it's just that the CI was killing `ld` on link because of lack of RAM. I'll change that back to see if it was just an issue on my end or if this org's CI will have the problem too

Reply:
j3 is still killed by CircleCI, maybe I broke this CI the same way I broke Appveyor, I'll check quickly